src/apis/exh/content/draft/_id@number.ts/publish/index.ts

```diff
@@ -0,0 +1,22 @@
+import { AuthToken } from "../../../../../@types";
+import { Draft } from "../../@types";
+
+export interface Methods {
+  /**
+   * draft を公開し、status を publish にする
```

Comment:
```suggestion
   * draft を公開し、status を published にする
```

先のれびゅーの方針とぶつかるけど、一応…